modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheRichTextWidgetEditor.vue

```diff
@@ -1,27 +1,42 @@
 <template>
-  <div class="apos-richtext-editor">
-    <editor-menu-bar :editor="editor">
-      <div class="apos-richtext-menubar" slot-scope="{ commands, isActive }">
-        <component
-          v-for="(item, index) in toolbar"
-          :key="item + '-' + index"
-          :is="(tools[item] && tools[item].component) || 'ApostropheTiptapUndefined'"
-          :name="item"
-          :tool="tools[item]"
-          :options="options"
-          :editor="editor"
-        />
-      </div>
-    </editor-menu-bar>
-    <editor-content :editor="editor" />
+  <div class="apos-rich-text-editor">
+    <component
+      :is="menuType"
+      :editor="editor"
+      :keep-in-bounds="false"
+      v-slot="{ commands, isActive, menu, focused }"
+    >
+      <AposContextMenuDialog
+        menu-placement="top"
+        class-list="apos-theme-dark apos-context-menu__dialog--unpadded apos-rich-text-editor__dialog apos-rich-text-toolbar"
```

Comment:
`apos-theme-dark apos-context-menu__dialog--unpadded` look like they should come from options passed in. At least the unpadded one.

Reply:
Removed instances of forcing a different theme on a specific app. This will break some design consistency with the art documents but will remove this anti pattern approach

Some long term solutions discussed:
- Themes come equipped with a fuller inverse/contrast-focused color set to use in these cases.
- Editors or developers can choose a different 'in-context' theme to handle UI overlaid on their project.

Follow up ticket created https://apostrophecms.atlassian.net/jira/software/projects/A30U/boards/4/backlog?selectedIssue=A30U-587